Makefile

```diff
@@ -278,11 +278,15 @@ out/doc/api/assets/%: doc/api_assets/% out/doc/api/assets/
 out/doc/%: doc/%
 	cp -r $< $@
 
+# check if ./node is actually set, else use user pre-installed binary
+gen-json = tools/doc/generate.js --format=json $< > $@
 out/doc/api/%.json: doc/api/%.md
-	$(NODE) tools/doc/generate.js --format=json $< > $@
+	[ $(NODE) ] && $(NODE) $(gen-json) || node $(gen-json)
```

Comment:
Didn't know about -x. Thought -n, -z or nothing at all would be appropriate . amended